pymc4/inference/utils.py

```diff
@@ -40,5 +40,4 @@ def trace_to_arviz(pm4_trace, pm4_sample_stats):
 
     posterior = {k: np.swapaxes(v.numpy(), 1, 0) for k, v in pm4_trace.items()}
     sample_stats = {k: v.numpy().T for k, v in pm4_sample_stats.items()}
-    sample_stats["tree_size"] = np.diff(sample_stats["tree_size"], axis=1)
```

Comment:
Just curious, what did this line, or not do because its being removed?

Reply:
Nevermind just connected with the comment up top. Thanks!

Reply:
@junpenglao Realized I'm still curious about this one, sorry. Curious on the background of this change from tfp. Is anything lost from removing `tree_size`? What was the benefit? Why the removal?

No code change required, just trying to learn